pkgs/applications/misc/remarkable/remarkable-mouse/default.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenv, buildPythonApplication, fetchPypi, pythonPackages }:
+
+buildPythonApplication rec {
+  pname = "remarkable-mouse";
+  version = "5.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1hlgv86r85l6bh90srphpkg4j70873mx3gq2bg0nc1m925qnw5wd";
+  };
+
+  propagatedBuildInputs = with pythonPackages; [ screeninfo paramiko pynput libevdev ];
+
+  meta = with stdenv.lib; {
+    description = "A program to use a reMarkable as a graphics tablet";
+    homepage = "https://github.com/evidlo/remarkable_mouse";
+    license = licenses.gpl3;
+    maintainers = [ maintainers.nickhu ];
+    platforms = platforms.all;
```

Comment:
You're sure about that? did you test it on Darwin?

Reply:
It's advertised to work on Mac, but I have no means of testing.

Reply:
Then please restrict it to linux for now.

Reply:
if tests were enabled, and they successfully ran on mac, then I would say it's fine

Reply:
Given how this software is just a python script, I think it's probably going to work fine on Mac, so I'll err to leaving it enabled